lib/postcode_validation/gateway/pca_address_list.rb

```diff
@@ -0,0 +1,43 @@
+module PostcodeValidation
+  module Gateway
+    class PCAAddressList
+      include HTTParty
+
+      class PCARequestError < PostcodeValidation::Error::RequestError; end
+
+      KEY = ENV['POSTCODE_ANYWHERE_KEY']
+      base_uri 'https://services.postcodeanywhere.co.uk'
```

Comment:
I personally think that there should only be one gateway/domain object. There is just a difference in the number of results